client/src/modules/fiscal/fiscal.js

```diff
@@ -25,17 +25,26 @@ function FiscalController($state, Fiscal, ModalService, Notify, $window) {
   vm.sortByDateCreated = 'ASC';
   vm.sortByNumberOfMonth = 'ASC';
 
+  vm.loadingState = true;
+  vm.loadingError = false;
+
+
   // refresh Fiscal Year
   function refreshFiscalYear() {
-    return Fiscal.read(null,{ detailed: 1 })
+    return Fiscal.read(null, { detailed : 1 })
       .then(function (fiscalYears) {
         vm.fiscalYears = fiscalYears;
+      }).catch(() => {
```

Comment:
Unfortunately, we can't use arrow functions on the client - they break our minified build.  Could you change all these to `function () {` calls?